### PR TITLE
TW-530 Partner Workspaces

### PR DESCRIPTION
**Pull request by claireriley**

### CODE REVIEW

File: src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+1. Under **Visibility**, select **Partner**.
+1. Select **Update**.
+
+Anyone with permissions to change workspace visibility can request to convert a workspace into a partner workspace, but it must be approved by a [Partner Manager](#managing-users-and-roles). If you're not a [Super Admin](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) or Partner Manager, Postman will mention this and you can add an optional note to your request. Postman will notify you when a Partner Manager reviews your request and approves or denies it.
+
+> When a workspace is converted to a partner workspace, Postman sends in-app and email notifications to all workspace members.
+
+## Setting up a partner workspace
+
+Before sharing a partner workspace with external collaborators, you can set your partnership up for success by moving relevant APIs, collections, and environments into it.
```
Comment (ArjunSingh-PM) [inline src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md:75]: The user can either move or fork the relevant element.

**Review by ArjunSingh-PM — APPROVED** [review]
Looks good. Please review comments from Ramji at https://postmanlabs.atlassian.net/wiki/spaces/TW/pages/4106387514/Partner+Workspaces

File: src/pages/docs/administration/managing-your-team/user-groups.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ Give your group a name and description, add your team members, and select the te
 
 > You must have the Billing role yourself to assign the Billing role to others.
 
-<img alt="Create a group form" src="https://assets.postman.com/postman-docs/create-a-group-9.5.jpg" width="500px"/>
+<img alt="Create a group form" src="https://assets.postman.com/postman-docs/v10/create-group1-v10.jpg" width="450px"/>
```
Comment (ArjunSingh-PM) [inline src/pages/docs/administration/managing-your-team/user-groups.md:54]: As of now, the Partner role is not supported for groups. We can keep the screenshot as is

File: src/pages/docs/administration/managing-your-team/user-groups.md [hunk 3]
```diff
@@ -67,7 +67,7 @@ In Postman, select **Team** in the upper-right corner > **Manage Team**. Under *
 
 Select **+ Add** to add a team member to the group. Select **X** next to a team member to remove them from the group.
 
-<img alt="Manage group members" src="https://assets.postman.com/postman-docs/edit-group-members-9.5.jpg" width="500px"/>
+<img alt="Manage group members" src="https://assets.postman.com/postman-docs/v10/edit-members-group1-v10.jpg" width="450px"/>
```
Comment (ArjunSingh-PM) [inline src/pages/docs/administration/managing-your-team/user-groups.md:70]: As of now, the Partner role is not supported for groups. We can keep the screenshot as is

File: src/pages/docs/administration/managing-your-team/user-groups.md [hunk 4]
```diff
@@ -81,7 +81,7 @@ You can control a group's access at the team level, workspace level, and on indi
 
 In Postman, select **Team** in the upper-right corner > **Manage Team**. Under **Members and groups**, select **Groups**. Locate the group you would like to update and select it to edit.
 
-<img alt="Edit group team roles" src="https://assets.postman.com/postman-docs/edit-group-roles-9.5.1.jpg" width="500px"/>
+<img alt="Edit group team roles" src="https://assets.postman.com/postman-docs/v10/edit-roles-group1-v10.jpg" width="450px"/>
```
Comment (ArjunSingh-PM) [inline src/pages/docs/administration/managing-your-team/user-groups.md:84]: As of now, the Partner role is not supported for groups. We can keep the screenshot as is

File: src/pages/docs/collaborating-in-postman/roles-and-permissions.md [hunk 5]
```diff
@@ -235,6 +227,28 @@ Run, pause, and resume monitor | &#x2714; |
...
+| **Workspace level** | Admin, Viewer, Editor | Viewer, Editor <br> Partner Lead (optional) |
+
+You can [assign](/docs/administration/managing-your-team/managing-your-team/) partner roles at the team level:
+
+* **Partner Manager** (Internal) - Manages all Partner Workspaces within an organization. Controls Partner Workspace settings and visibility, and can send invites to partners.
+* **Partner** (External) - All partners are automatically granted the Partner role at the team level. Partners can only access the Partner Workspaces they've been invited to.
+
+You can also [assign](/docs/collaborating-in-postman/using-workspaces/managing-workspaces/#managing-workspace-roles) partner roles at the workspace level:
+
+* **Partner Lead** (External) - Can invite other partners from their organization to join a Partner Workspace.
```
Comment (ArjunSingh-PM) [inline src/pages/docs/collaborating-in-postman/roles-and-permissions.md:248]: Partner Lead is also a workspace-level role. IMO, "workspace roles" be a better place to explain this.

**Review by ArjunSingh-PM — COMMENTED** [review]
@claireriley, Partner role is not supported for groups. Let us keep the screenshots as is for groups.

File: src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+
+You must be an Editor on an API, collection, or environment, the Workspace Admin, or a Super Admin to remove an element from a Partner Workspace.
+
+To remove an element by moving it to another workspace:
+
+1. In the sidebar, select the element you want to move, then select the more actions icon <img alt="More actions icon" src="https://assets.postman.com/postman-docs/icon-more-actions-v9.jpg#icon" width="16px"> > **Move**.
+
+1. Use the search bar to find the workspace you'd like to move the element to, or filter by workspace visibility by selecting the filter icon <img alt="Filter icon" src="https://assets.postman.com/postman-docs/icon-filter.jpg#icon" width="16px">.
+
+    > You can move elements to workspaces that you have access to.
```
Comment (HeatherStenson) [inline src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md:170]: maybe:

```suggestion
    > You can only move elements to workspaces that you have access to.
```

**Review by HeatherStenson — APPROVED** [review]
You distilled a lot of complex info into a very manageable format, bravo! 👏

File: src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md [hunk 7]
```diff
@@ -0,0 +1,187 @@
...
+
+1. Define a workspace **Name** and **Summary**.
+1. Under **Visibility**, select **Partner**.
+1. Select **Create workspace**.
+
+    <img alt="Create Partner Workspace" src="https://assets.postman.com/postman-docs/v10/create-partner-workspace-v10.jpg" width="400px"/>
+
+Anyone with permissions to create workspaces can request to create a Partner Workspace, but it must be approved by a [Partner Manager](#managing-users-and-roles). If you're not a [Super Admin](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) or Partner Manager, you must submit your request for approval. You can also add an optional note to your request. The workspace's visibility will be set to [private](/docs/collaborating-in-postman/using-workspaces/managing-workspaces/#changing-workspace-visibility) until your request is approved. Postman will notify you when a Partner Manager reviews your request and approves or denies it.
+
+<img alt="Create Partner Workspace approval" src="https://assets.postman.com/postman-docs/v10/partner-workspace-approval-warning-v10.jpg" width="450px" />
```
Comment (ramji367) [inline src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md:46]: The text shown in this image has change. Can you take a new screenshot?

File: src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md [hunk 8]
```diff
@@ -0,0 +1,187 @@
...
+1. Open the workspace and select **Invite** in the upper-right corner.
+1. (Workspace Admins and Partner Managers only) Select **Partners**, then **Continue**.
+1. Enter the name, email, or group name of the partners.
+1. Set their [workspace role](/docs/collaborating-in-postman/roles-and-permissions/#workspace-roles).
+1. (Workspace Admins and Partner Managers only) Select **Make them Partner Lead** if you'd like them to also be able to manage and invite partners within the workspace.
+1. Select **Invite Partners**.
+
+Partner invites must be accepted by the email ID you provide when inviting the individual. If a partner would like to use a different email ID, or if an invite was mistakenly sent, you can [revoke the invite](/docs/administration/managing-your-team/managing-your-team/#managing-invitations) and send a new one.
+
+> Partners will sign in to your team with a Postman password, even if your team is restricted to signing in with [SSO](/docs/administration/sso/intro-sso/).
```
Comment (ramji367) [inline src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md:97]: Should we say "...with a Postman _username and_ password"? I suppose let's be consistent in language with other LC docs.

File: src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md [hunk 9]
```diff
@@ -0,0 +1,187 @@
...
+
+Workspace Admins and Partner Managers can invite team members to Partner Workspaces.
+
+To invite team members to a Partner Workspace:
+
+1. Open the workspace and select **Invite** in the upper-right corner.
+1. Select **Team members**, then **Continue**.
```
Comment (ramji367) [inline src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md:104]: Only workspace admins and Partner managers will see the screen in (2). Other users will directly jump to (3)

File: src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md [hunk 10]
```diff
@@ -0,0 +1,187 @@
...
+
+Within a Partner Workspace, team members can have different workspace roles: Admin, Editor, and Viewer. To learn more, see [Workspace roles](/docs/collaborating-in-postman/roles-and-permissions/#workspace-roles).
+
+Team members can also be assigned the team-level Partner Manager role. Partner Managers can view all Partner Workspaces within an organization, and can control their settings, visibility, and partner invites. To learn more, see [Managing users and roles](#managing-users-and-roles).
+
+### Collaborating as a partner
+
+When an external partner is invited to a Partner Workspace by a team they're working with, they only have access to that Partner Workspace and can't access any other workspaces or resources within the team. Partners don't have personal or team workspaces within the team they're partnering with.
+
+Within a Partner Workspace, partners can [import and export](/docs/getting-started/importing-and-exporting-data/) elements. They can also [fork elements](/docs/collaborating-in-postman/version-control/#forking-postman-entities) from the Partner Workspace to another workspace, even if it's outside of your team.
```
Comment (ramji367) [inline src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md:125]: This needs to be modified a little bit: If the partner is an Editor in a Partner Workspace, they can import and export elements. Editors can also fork elements within the same Partner Workspace or to a different Partner Workspace within the same team. They can't fork elements outside the team

File: src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md [hunk 11]
```diff
@@ -0,0 +1,187 @@
...
+
+At the team level, all external partners are automatically assigned the Partner role. At the workspace level, all partners are assigned [Workspace Editor or Viewer](/docs/collaborating-in-postman/roles-and-permissions/#workspace-roles) roles when invited to a Partner Workspace. Partners can also be assigned the Partner Lead role, which enables them to invite other individuals from their organization to the Partner Workspace.
+
+> To learn more about partner roles and permissions, see [Partner Roles](/docs/collaborating-in-postman/roles-and-permissions/#partner-roles).
+
+## Managing a Partner Workspace
+
+Workspace Admins and Partner Managers manage workspace settings, members, and roles within a Partner Workspace. Partner Leads also manage members and roles in Partner Workspaces.
```
Comment (ramji367) [inline src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md:133]: Partner Leads manage members and roles _of partners_ in the Partner Workspace

File: src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md [hunk 12]
```diff
@@ -0,0 +1,187 @@
...
+
+> When a workspace is converted to a Partner Workspace, Postman sends in-app and email notifications to all workspace members.
+
+## Setting up a Partner Workspace
+
+Before sharing a Partner Workspace with external collaborators, you can set your partnership up for success by moving or forking relevant elements into it.
+
+### Moving elements to a Partner Workspace
+
+You must be an Editor on an API, collection, or environment, the Workspace Admin, or a Super Admin to move an element to a Partner Workspace. To move an element:
```
Comment (ramji367) [inline src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md:64]: Partner Manager can also move or delete any element within a partner workspace

File: src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md [hunk 13]
```diff
@@ -0,0 +1,187 @@
...
+
+1. Select the workspace, then **Move** (**Collection**, **API**, or **Environment**).
+
+It's not recommended, but you can choose to delete an element entirely. To do so, select the element in the sidebar, then select the more actions icon <img alt="More actions icon" src="https://assets.postman.com/postman-docs/icon-more-actions-v9.jpg#icon" width="16px"> > **Delete**.
+
+> Deleted APIs and environments aren't recoverable. You can recover collections smaller than 30 MB for up to 90 days on an Enterprise plan. To do so, select <img alt="Delete icon" src="https://assets.postman.com/postman-docs/icon-delete-v9.jpg#icon" width="12px"> **Trash** from the Postman footer.
+
+## Deleting a Partner Workspace
+
+Workspace Admins and Partner Managers can delete Partner Workspaces. To delete a Partner Workspace:
```
Comment (ramji367) [inline src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md:180]: When a Partner Manager or Workspace admin tries to change visibility, we provide a checkbox to remove all the partners in the workspace at once. They can do this and then process to delete the workspace. Loom recording: https://www.loom.com/share/072b91194fc943a58d2ba8cc5d7e7def

File: src/pages/docs/collaborating-in-postman/roles-and-permissions.md [hunk 14]
```diff
@@ -107,11 +101,12 @@ To learn how to manage team roles in Postman, see [Managing your team](/docs/adm
 
 ## Workspace roles
 
-You can [assign](/docs/collaborating-in-postman/using-workspaces/managing-workspaces/#managing-workspace-roles) three role types in Postman workspaces: **Admin**, **Editor**, and **Viewer**.
+You can [assign](/docs/collaborating-in-postman/using-workspaces/managing-workspaces/#managing-workspace-roles) three role types in Postman workspaces: **Admin**, **Editor**, and **Viewer**. Partner Workspaces offer an additional role type: **Partner Lead**.
 
 * **Admin** - Can manage workspace resources and settings
 * **Editor** - Can create and edit workspace resources
 * **Viewer** - Can view, fork, and export workspace resources
+* **Partner Lead** (External, [Enterprise plans only](https://www.postman.com/pricing)) - Can invite other partners from their organization to join a [Partner Workspace](/docs/collaborating-in-postman/using-workspaces/partner-workspaces/). To learn more, see [Partner roles](#partner-roles).
```
Comment (ramji367) [inline src/pages/docs/collaborating-in-postman/roles-and-permissions.md:109]: Internal/External seems hard to understand for me. Maybe we have used it in other docs so it is actually consistent? The other thing is I'm not sure if we should call out Partner Lead as a "role". Technically it is an "additional permission" that a partner user can have. But that's not easy to communicate. Something to think about it - I'll let you take the call on this

**Review by ramji367 — APPROVED** [review]

File: src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md [hunk 15]
```diff
@@ -0,0 +1,187 @@
+---
+title: "Partner workspaces"
+order: 76
+page_id: "partner_workspaces"
+updated: 2022-08-29
```
Comment (jkonrath-postman) [inline src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md:5]:
```suggestion
updated: 2022-09-15
```

**Review by jkonrath-postman — APPROVED** [review]
LGTM, one date flip but otherwise great!

**Pull request merged**